src/tracer.js

```diff
@@ -251,6 +258,7 @@ function getTrimmedTrace(traceSize, jsTrace) {
  */
 function sendCurrentTrace(traceSender) {
     const tracerObj = module.exports.getTrace();
+    setLabelsToEvent(tracerObj)
```

Comment:
I would move this and the next one a few lines below, since tracerObj can be `undefined`